internal/cache/file/downloader/parallel_downloads_job_test.go

```diff
@@ -33,10 +33,12 @@ type parallelDownloaderTest struct {
 	downloaderTest
 }
 
-func init() { RegisterTestSuite(&parallelDownloaderTest{}) }
+func init() {
+	RegisterTestSuite(&parallelDownloaderTest{})
+}
 
 func (dt *parallelDownloaderTest) SetUp(*TestInfo) {
 	dt.defaultFileCacheConfig = &config.FileCacheConfig{EnableParallelDownloads: true,
-		DownloadParallelismPerFile: 5, ReadRequestSizeMB: 2, EnableCrcCheck: true}
+		DownloadParallelismPerFile: 5, ReadRequestSizeMB: 2, EnableCrcCheck: true, MaxDownloadParallelism: 6}
```

Comment:
I think MaxDownloadParallelism doesn't matter here since we are anyways passing semaphore.NewWeighted(math.MaxInt64) in job_test.go. so, would recommend removing this field from here to avoid confusion in the past.